drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -189,6 +191,23 @@ def self.summary_field(name, type = nil, **kwargs)
 
     field :move_in_addresses, [HmisSchema::ClientAddress], null: false
 
+    EXCLUDED_HISTORY_FIELDS = ['id', 'DateCreated', 'DateUpdated', 'DateDeleted'].to_set.freeze
```

Comment:
Should we note that other excluded fields would need to be mountain case, or if that's not a requirement, maybe we want to snake case these?

Reply:
the fields should match our DB casing, so they will be inconsistent. I'll add a comment